bigtable/tests/unit/test_table.py

```diff
@@ -411,6 +411,48 @@ def _list_column_families_helper(self):
     def test_list_column_families(self):
         self._list_column_families_helper()
 
+    def test_get_cluster_states(self):
+        from google.cloud.bigtable_admin_v2.gapic import (
+            bigtable_table_admin_client)
+        from google.cloud.bigtable.enums import Table as enum_table
+        from google.cloud.bigtable.table import _ClusterState
+        INITIALIZING = enum_table.ReplicationState.INITIALIZING
+        PLANNED_MAINTENANCE = enum_table.ReplicationState.PLANNED_MAINTENANCE
+        READY = enum_table.ReplicationState.READY
+
+        table_api = bigtable_table_admin_client.BigtableTableAdminClient(
+            mock.Mock())
+        credentials = _make_credentials()
+        client = self._make_client(project='project-id',
+                                   credentials=credentials, admin=True)
+        instance = client.instance(instance_id=self.INSTANCE_ID)
+        table = self._make_one(self.TABLE_ID, instance)
+
+        response_pb = _TablePB(
+            cluster_states={'cluster-id1': _ClusterStatePB(INITIALIZING),
```

Comment:
Use `_ClusterState` here, too.

Reply:
@tseaver 

I have tried with new class _ClusterState. Unit test fail for it reporting as:

    def MergeFrom(self, msg):
      if not isinstance(msg, cls):
        raise TypeError(
            "Parameter to MergeFrom() must be instance of same class: "
>           'expected %s got %s.' % (cls.__name__, msg.__class__.__name__))
E       TypeError: Parameter to MergeFrom() must be instance of same class: expe
cted ClusterState got _ClusterState.

e:\venv\scripts\.nox\unit-2-7\lib\site-packages\google\protobuf\internal\python_
message.py:1230: TypeError

Any suggestion on this.

Reply:
Here we need a protobul `ClusterState` object to mock the return from the server

Reply:
We can import Table from `table_pb2`

```python
from google.cloud.bigtable_admin_v2.proto.table_pb2 import Table
```
And then build the `reponse_pb` as follows

```python
        response_pb = _TablePB(
            cluster_states={
                'cluster-id1': Table.ClusterState(
                    replication_state=INITIALIZING),
                'cluster-id2': Table.ClusterState(
                    replication_state=PLANNED_MAINTENANCE),
                'cluster-id3': Table.ClusterState(
                    replication_state=READY),
            },
        )
```

This way we can remove `def _ClusterStatePB` from `test_table`